### PR TITLE
E2e test fix

### PR DESCRIPTION
**Pull request by lavalamp**
These two e2e tests seem totally broken due to not creating their namespace(s). This fixes them.

Needed for #8155.

### CODE REVIEW

**Comment by thockin** [comment]
Services was already fixed, sorry

**Comment by bgrant0607** [comment]
Are the test/e2e/es_cluster_logging.go changes needed to fix e2e?

**Comment by bgrant0607** [comment]
Actually, services.sh is still failing.

**Comment by lavalamp** [comment]
@bgrant0607 yes, both of these tests are broken at head.

@thockin It's not fixed at head? Is there another PR out?

**Comment by bgrant0607** [comment]
I reverted both the fix (#9141) and the original PR (#8953).

File: hack/e2e-suite/services.sh [hunk 1]
```diff
@@ -34,6 +34,24 @@ source "${KUBE_VERSION_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
...
+${KUBECTL} create -f - << __EOF__
+{
+    "kind": "Namespace",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "${NAMESPACE}"
+    },
+    "spec": {
+        "finalizers": [
+            "kubernetes"
```
Comment (bgrant0607) [inline hack/e2e-suite/services.sh]: I don't think this finalizer is needed for a namespace other than default.

Comment (lavalamp) [inline hack/e2e-suite/services.sh]: I don't know what this does--I'm just copying what our other test namespaces look like. I can remove it if you want.

Comment (thockin) [inline hack/e2e-suite/services.sh]: I cannot repro the failure, so I don't know how to fix it properly.  I have
a PR with the original test fix and the not-fixed fix fix.  It works for
me, but not in jenkins.  I don't know how to fix.

Comment (bgrant0607) [inline hack/e2e-suite/services.sh]: Yes, please remove the finalizer.

Comment (bgrant0607) [inline hack/e2e-suite/services.sh]: This PR fixes one obvious problem -- the non-existent and unnecessary namespace shouldn't be specified when creating the namespace. That generates an error, if nothing else.

**Comment by lavalamp** [comment]
OK, stuff moves far too quickly around here-- now the change to services.sh here is just in case someone gets fancy and wants to try again with adding a namespace to the test. The other change is still needed AFAIK.

File: hack/e2e-suite/services.sh (hunk 1 above)
Comment (thockin) [inline hack/e2e-suite/services.sh]: I'm so confused - I can not reproduce the error?!?!

Comment (thockin) [inline hack/e2e-suite/services.sh]: Also this PR will not apply with out a revert of the revert of the initial
breaking change

**Comment by thockin-cc** [comment]
Daniel pointed out that I built but did not push.  Is it Friday yet?

**Comment by thockin-cc** [comment]
I'll have a full fix for services.sh momentarily.

File: hack/e2e-suite/services.sh (hunk 1 above)
Comment (derekwaynecarr) [inline hack/e2e-suite/services.sh]: It is needed but it's applied on the server if omitted on create

**Comment by bgrant0607** [comment]
@lavalamp Please remove the services test fix and I'll merge the elasticsearch fix separately.

**Comment by lavalamp** [comment]
@bgrant0607 Removed.

**Comment by bgrant0607** [comment]
Thanks. LGTM. Rerunning shippable.

**Comment by bgrant0607** [comment]
Hmm. Shippable failed again. Trying one more time. If this doesn't work, I'm going to ask you to rebase and try again.

**Comment by lavalamp** [comment]
Shippable happy now for whatever reason.

**Pull request merged**